common/linux/build.sh

```diff
@@ -23,6 +18,11 @@ builder_describe "Keyman common Linux modules" \
 
 builder_parse "$@"
 
+if [[ $BUILDER_OS != linux ]]; then
```

Comment:
A builder script must always call `builder_describe` and `builder_parse`, so do our platform checks after that. This allows the script to be called with `--help` even when on the wrong platform, so user can get info about it.